coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 97.54
+    "covered_percent": 95.41
```

Comment:
Some of our test files are spread out across different branches so I think when we merge that should solve the coverage problems